routes/web.php

```diff
@@ -186,6 +186,7 @@
         Route::group(['prefix' => 'me'], function () {
             Route::put('', "UserApiController@updateMe");
             Route::put('pic',  "UserApiController@updateMyPic");
+            Route::get('actions', "UserActionApiController@getActionsByCurrentUser");
         });
 
         Route::get('actions', "UserActionApiController@getActions");
```

Comment:
this is missing the proper middleware please check the ticket

Reply:
done